### ISSUE TITLE
aws_api_gateway_integration request_parameters JSON?!

### ISSUE THREAD

**Issue by ianaz** [hashicorp/terraform#10223]
Hi there,

More examples for the "request_parameters" would be veeery useful :) maybe also how to send a custom json object to the lambda. I'm probably doing something wrong but I can't bring it to work

### Terraform Version
v0.7.11

### Affected Resource(s)
- aws_api_gateway_integration

### Terraform Configuration Files
```hcl

resource "aws_api_gateway_integration" "community_integration" {
  rest_api_id = "${var.apigateway_id}"
  resource_id = "${aws_api_gateway_resource.api_gateway_res_community_comments.id}"
  http_method = "${aws_api_gateway_method.method_get_comments.http_method}"
  credentials = "${var.lambdarole_arn}"

  type = "AWS"
  integration_http_method = "POST"
  uri = "arn:aws:apigateway:${var.region}:lambda:path/2015-03-31/functions/arn:aws:lambda:${var.region}:${var.account_id}:function:${aws_lambda_function.lambda_community.function_name}/invocations"
  depends_on = [
    "aws_api_gateway_method.method_get_comments"]
request_parameters = { 
"integration.request.header.X-Some-Other-Header" = "method.request.header.X-Some-Header"
 }
}
```

### Debug Output
https://gist.github.com/ianaz/80891b04b939aa16611ea45f4b4649e1

### Expected Behavior
I expect Terraform to create the integration request

### Actual Behavior
I get the error specified

Even by following [your example ](https://www.terraform.io/docs/providers/aws/r/api_gateway_integration.html#request_parameters) I get this error.

**Comment by ianaz** [comment]
Had to fix with

request_parameters {
    "integration.request.header.X-Some-Other-Header" = "'method.request.header.X-Some-Header'"
  }

(note the double quote + single quote). 
Definitely a bug

**Comment by stack72** [comment]
Closed via #13249
